mage_ai/io/duckdb.py

```diff
@@ -89,10 +89,7 @@ def get_type(self, column: Series, dtype: str) -> str:
             PandasTypes.UNKNOWN_ARRAY,
             PandasTypes.COMPLEX,
         ):
-            raise BadConversionError(
-                f'Cannot convert column \'{column.name}\' with data type \'{dtype}\' to '
-                'a DuckDB datatype.'
-            )
+            return 'text'
```

Comment:
Should it use upper case?

Reply:
I tried so both upper case and lower case works. But I update here to be upper case to be consistent with other type string.